src/hub.ts

```diff
@@ -372,15 +372,6 @@ export class Hub extends EventEmitter {
         return port;
     }
 
-
-    protected _lpad (str: string, length: number) {
```

Comment:
This function is unused since the version parsing changes, but I used it in my fork for the previous version of `toBin` and `toHex`.